app/Console/Commands/ImportMfolGroupsCommand.php

```diff
@@ -79,10 +79,10 @@ public function handle()
             } catch (Exception $e) {
                 $numFailed++;
 
-                info('Error importing group with ' .$name . ':' . $e->getMessage());
+                info('Error importing group with '.$name.':'.$e->getMessage());
```

Comment:
@aaronschachter per your [comment](https://github.com/DoSomething/rogue/pull/1090#discussion_r466743200), this PHP Style Fixer tool will fix all of these instances based on the specified rules! 

I'll also add it to the communal docs, cause it's good practice, but wanted to highlight this 😄